src/styles/measure.css

```
@@ -1,22 +1,63 @@
+    min(
+      calc(100vw / 1512px),
+      calc(100dvh / 982px)
+    )
+  );
 
-  /* Frame 53 と同系：#13A2D7 → #F2EDCC */
-  background: linear-gradient(180deg, #13a2d7 0%, #f2edcc 100%);
+  background-color: #f3f3f3;
+  background-image: linear-gradient(180deg, #13a2d7 0%, #f2edcc 100%);
 
   display: flex;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)